packages/product-service/src/handlers/getProductsList.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { APIGatewayProxyEvent } from "aws-lambda";
+import { getProducts } from "src/services";
+
+import { getProductsListHandler } from "./getProductsList";
+
+jest.mock("src/services", () => ({
+  getProducts: jest.fn(),
+}));
+
+describe("getProducts", () => {
+  it("should return 200 response and the products", async () => {
+    const event: APIGatewayProxyEvent = {
+      queryStringParameters: {},
+    } as any;
+
+    (getProducts as jest.Mock).mockResolvedValueOnce([
+      { title: "test-product" },
+    ]);
+
+    const resp: any = await getProductsListHandler(event, null, null);
```

Comment:
I'd test not the handler (infrastructure logic), but focus on the underlying service that does business or application logic (validation, fetching from db, remapping...)